squiRL/common/agents.py

```diff
@@ -87,13 +82,13 @@ def play_step(
             done (bool): Indicates if a step is terminal
         """
         action = self.get_action(net)
+        action = [action] if type(action) == int else action
```

Comment:
Please see my comment above. This is the redundant line I'm referring to.

Reply:
Done.